.travis.yml

```diff
@@ -6,6 +6,8 @@ python:
     - "2.7"
     - "2.6"
 install:
+    - if [[ $TRAVIS_PYTHON_VERSION == 2.6 ]]; then pip install mock; fi
+    - if [[ $TRAVIS_PYTHON_VERSION == 2.7 ]]; then pip install mock; fi
```

Comment:
Ah, I misread that a bit. You could simplify this a little bit to `if [ "$TRAVIS_PYTHON_VERSION" = 2.6 -o  "$TRAVIS_PYTHON_VERSION" = 2.7 ] ; then ... ; fi`